packages/terra-clinical-result/src/resulttimeheadercell/ResultTimeHeaderCell.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './ResultTimeHeaderCell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content to be displayed on the first line, typically abbrivated date, e.g. `Dec 12, 2010`
+   */
+  date: PropTypes.string,
+  /**
+   * Content to be displayed on the second line, typically 24 hour time in hours and minutes, e.g. `23:59`
+   */
+  time: PropTypes.string,
+  /**
+   * Visually hides the date when presented in a series of side-by-side columns of the same date.
```

Comment:
I see this is ensure the alignment is correct but is this value read by screen readers or why not supply an empty date string instead? or no supply a date and always render the data div with min-height?